ccc/ccc.go

```diff
@@ -49,7 +49,9 @@ var sortingRegexp = regexp.MustCompile("_(.+)")
 // Simple handler to handle fatal errors
 func fatal(err error) {
 	_, _ = fmt.Fprintf(os.Stderr, "%s: %s\n", os.Args[0], err)
-	os.Exit(1)
+	if _, exists := os.LookupEnv("CONTINUE_ON_ERROR"); ! exists {
+        os.Exit(1)
```

Comment:
Still an indent missing here. :)

Reply:
Note to self: Don't copy examples with tabs 🤦‍♂️